dist/index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Leaderboard</title>
+<script defer src="main.js"></script></head>
+<body class="container">
+  <h1 class="my-5">Leaderboard</h1>
+  <main class="row">
+    <section id="recent-scores" class="col-md-6 px-3">
+      <div id="recent-scores-head" class="d-flex justify-content-around">
+        <h2>Recent Scores</h2>
+        <button class="my-auto btn btn-primary">Refresh</button>
+      </div>
+      <div id="recent-scores-body" class="px-md-3 mt-3">
+        <table class="table table-dark table-sm table-striped table-hover">
+          <thead>
+            <td>User</td>
+            <td>Score</td>
+          </thead>
+          <tbody>
+            <tr>
+              <td>Name</td>
+              <td>100</td>
+            </tr>
+            <tr>
+              <td>Name</td>
+              <td>20</td>
+            </tr>
+            <tr>
+              <td>Name</td>
+              <td>50</td>
+            </tr>
+            <tr>
+              <td>Name</td>
+              <td>78</td>
+            </tr>
+            <tr>
+              <td>Name</td>
+              <td>125</td>
+            </tr>
+          </tbody>
+        </table>
```

Comment:
- [ ] It would be good if you populate the scores table with dynamically generated DOM elements. You can have a hardcoded scores list. Also, don't forget to use arrow functions in your ES6 JS modules. :pray: The wireframe looks fantastic though. :+1: :clap:

Reply:
Hi @henatan99 :wave:
Please, checkout the requirements for this milestone:
> ### Project requirements
> 
> - **IMPORTANT NOTE** This is the first time you will use [Gitflow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/gitflow.md). Keep that in mind!
> - Create an **npm project with webpack**. (If you need a refresher, check the previous project or webpack exercise).
> - Write plain HTML markup with minimum styling (just to make the layout), because you will implement styles in the following steps. Use this wireframe as a guide:
> 
> <p align="center">
>   <img src="https://raw.githubusercontent.com/microverseinc/curriculum-javascript/main/leaderboard/images/leaderboard_wireframe.png?token=AP4LXCTEKVOZ3TO4ERA5AXDBG7T3Q" alt="Basic UI"  width="500px"/>
> </p>
> 
> - Don't make API calls (you'll do it in a following lesson).
> - Use ES6 modules, with *import* and *export*.

The table rows you're referring to are to be fetched from the API, so, don't worry, they will be added dynamically, at the end of this project. This is just the first milestone, though. **Can you please reconsider your review?**

Reply:
@enionsouza, I am not saying you need to fetch data from API, but as the table data represent data to be fetched in the next milestone, you can have a hardcoded list (The list of rows you have in your HTML) of scores objects which you can populate by JS DOM manipulation. Sorry if I made any confusion.